shiny/api-examples/chat_bookmark/app-express.py

```diff
@@ -0,0 +1,93 @@
+# ------------------------------------------------------------------------------------
+# A basic Shiny Chat example powered by OpenAI.
+# ------------------------------------------------------------------------------------
+
+import requests
+from chatlas import ChatOpenAI, Turn
+from chatlas._content import Content
+from dotenv import load_dotenv
+
+from shiny import reactive
+from shiny.bookmark import BookmarkState
+from shiny.express import app_opts, session, ui
+from shiny.types import MISSING
+
+with ui.hold():
+    load_dotenv()
+
+chat_client = ChatOpenAI(model="gpt-4o-mini")
+
+
+# Set some Shiny page options
+ui.page_opts(
+    fillable=True,
+    fillable_mobile=True,
+)
+
+# Create and display a Shiny chat component
+chat = ui.Chat(
+    id="chat",
+)
+chat.ui(messages=["Hello! Would you like to know the weather today?"])
+
+
+chat.enable_bookmarking(
+    chat_client,
+    bookmark_on="response",
+    # ONLY for ChatExpress.
+    # For shiny-core `ui.Chat()``, use the `App(bookmark_store=)` directly
+    bookmark_store="url",
+)
```

Comment:
Will this example show up in the API reference somewhere?

FWIW, I think it's more important that we include `chat.enable_bookmarking()` in the all the templates under `shiny/templates/chat/*` rather than having an additional example/template.

Also, once we have the story for displaying tool calls, then I'd like to add tool calling templates, but not until next release.

Reply:
> Will this example show up in the API reference somewhere?

It should given `ui.Chat` is in both express and core docs.

----------------

Not resolving until templates are updated